retail/solutions/frontline-worker/README.md

```diff
@@ -0,0 +1,145 @@
+# Real-time store communications and collaboration
+
+## Table of Contents
+
+- [Microsoft Teams Integration with D356 Commerce](#microsoft-teams-integration-with-d365-commerce)
+  - [Table of Contents](#table-of-contents)
+
+
+## Microsoft Teams Integration with D356 Commerce
+
+[Dynamics 365 Commerce is integrating with Microsoft Teams](https://docs.microsoft.com/en-us/dynamics365/teams-integration/teams-integration) to help customers and their employees improve productivity by synchronizing task management between the two applications. The seamless task management that Commerce and Teams integration provides let store managers and employees create task lists, assign tasks to multiple stores, and track the status of tasks across stores, from either application.
+
+### Capabilities
+
+
+Here are some of the key features that the Commerce and Microsoft Teams integration provides:
```

Comment:
Reword? 
"Key features of D365 Commerce and Microsoft Teams are:"

Reply:
Perhaps, also add a link to MS Docs which elaborates on these points and readers can go through those details if they want to go deeper?